Lib/test/test_import/__init__.py

```diff
@@ -163,10 +163,7 @@ def test_import(self):
         def test_with_extension(ext):
             # The extension is normally ".py", perhaps ".pyw".
             source = TESTFN + ext
-            if is_jython:
-                pyc = TESTFN + "$py.class"
-            else:
-                pyc = TESTFN + ".pyc"
+            pyc = TESTFN + ".pyc"
```

Comment:
I am removing Jython compat from tests, because there are lots and lots of new tests since 2.7

New syntax, new corner cases, etc.

If Jython ever run our new suite, there will be a lot of things to fix / adapt on their side.